### PR TITLE
Adds docs for wallet

### PR DESCRIPTION
**Pull request by AlbertoElias**

### CODE REVIEW

**Review by druiz0992 — DISMISSED** [review]
I added a link to the sidebar

File: docs/developers/hermez-wallet.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# Hermez Wallet
+
+## What is Hermez Wallet
+
+It is the first wallet to interact with the Hermez network. It supports depositing, transferring and withdrawing ETH and ERC-20 tokens in Hermez. Wallet provides a simple UI to get started.
```
Comment (jleebee) [inline docs/developers/hermez-wallet.md]: "It is the first wallet to interact with the Hermez Network." 

"The wallet provides a simple UI to get started" 
or 
"Hermez Wallet provides a simple UI to get started."

File: docs/developers/hermez-wallet.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+They all follow a similar flow. First, we select a token. If it's a `Deposit`, the token must be in the Ethereum account. Otherwise, it must be in the Hermez account.
+
+![](wallet/hw-deposit-accounts.png)
+
+Then there's a form to select the amount:
+
+![](wallet/hw-deposit-form.png)
+
+In case of a `Transfer`, there will also be a **receiver** input. This input also supports scanning a QR code or pasting the Receiver's address directly.
```
Comment (jleebee) [inline docs/developers/hermez-wallet.md]: "In the case of a..."

File: docs/developers/hermez-wallet.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+Making `Deposits` creates accounts that now appear in the home screen.
+
+![](wallet/hw-home.png)
+
+Opening an account shows all the transactions related to that account.
+
+![](wallet/hw-account.png)
+
+And opening a transaction shows information in regard to that transaction. There's also a button to open the Batch Explorer with all the information.
```
Comment (jleebee) [inline docs/developers/hermez-wallet.md]: "Opening a transaction shows information related to that transaction."

File: docs/developers/hermez-wallet.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+![](wallet/hw-tx.png)
+
+## Withdraws
```
Comment (jleebee) [inline docs/developers/hermez-wallet.md]: Withdraw or Withdrawals

File: docs/developers/hermez-wallet.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+They are a 2 part process. After doing the first part as explained above, a card appears in the Home or in the respective account page. When ready, it will show a button to finalise the withdrawal.
```
Comment (jleebee) [inline docs/developers/hermez-wallet.md]: "Withdrawals are a two-part process.  After completing the first part explained above, a card appears on the Home screen or on the respective account page."

**Review by jleebee — CHANGES_REQUESTED** [review]

**Comment by AlbertoElias** [comment]
@jleebee Thank you! Fixed!

**Pull request merged**